app/models/direct_assessment_form.rb

```diff
@@ -0,0 +1,72 @@
+class DirectAssessmentForm
+  include ActiveModel::Model
+
+  attr_accessor :outcome_ids, :direct_assessment
+  delegate :persisted?, to: :direct_assessment
+
+  validate :ensure_single_department
+
+  def self.model_name
+    DirectAssessment.model_name
+  end
+
+  def self.from(assessment)
+    new(assessment, assessment.outcome_assessments.pluck(:outcome_id))
+  end
+
+  def initialize(direct_assessment, outcome_ids)
+    @direct_assessment = direct_assessment
+    @outcome_ids = Array.wrap(outcome_ids).reject(&:blank?)
```

Comment:
This is a good call -- both `attr_accessor`s in the form object can actually be readers.